scripts/west_commands/completion/west-completion.bash

```diff
@@ -1142,6 +1142,52 @@ __comp_west_twister()
 	esac
 }
 
+__comp_west_sdk()
+{
+	local bool_opts="
+		--interactive -i
+		--no-toolchains -T
+		--no-hosttools -H
+	"
+
+	local dir_opts="
+		--install-dir -d
+	"
+
+	local other_opts="
+		--version -v
```

Comment:
As short form has been removed, then this should be updated.
```suggestion
		--version
```

Same probably goes for other completion handlers.

Btw, thank for providing completion handlers from the start (y)

Reply:
> As short form has been removed, then this should be updated.

I forgot about this one too. Thank you for pointing out.

> Btw, thank for providing completion handlers from the start 👍
I have finished bash and fish, but zsh requires some consideration, so I only included the bare minimum.
I'll update this soon.